static/js/app.js

```diff
@@ -178,21 +178,21 @@ endeavorForm?.addEventListener('submit', (event) => {
         setTimeout(() => {
           endeavorConfirmation.classList.add('hidden');
         }, 5000);
+      } else {
```

Comment:
This was in the wrong spot and has been moved so that it appears if a `204` isn't returned.